batch/create-batch-tables.sql

```diff
@@ -56,11 +56,11 @@ DELIMITER $$
 CREATE TRIGGER trigger_jobs_insert AFTER INSERT ON jobs
     FOR EACH ROW BEGIN
         UPDATE batch SET n_jobs = n_jobs + 1 WHERE id = new.batch_id;
-        IF (NEW.state LIKE 'Complete' OR NEW.state LIKE 'Cancelled') THEN
+        IF (NEW.state LIKE 'Error' OR NEW.state LIKE 'Failed' OR NEW.state LIKE 'Success' OR NEW.state LIKE 'Cancelled') THEN
```

Comment:
As a total aside, I find it amazing that SQL semantics has "=" as less exact than "LIKE", in breaking with I think all natural languages.